action.yaml

```diff
@@ -11,6 +11,12 @@ inputs:
       are not supported, because partial cache restoration leads to a "snowball"
```

Comment:
Nit (commit message): the scope should be "action" rather than "ci" since "ci" within this repository would refer to the CI pipeline for this action itself. (In fairness, this action is run as part of its own CI... he he.)